examples/sorting/Selection Sort/Go/main.go

```diff
@@ -0,0 +1,42 @@
+// Selection sort
+//
+// Time complexity: O(N^2)
+//   - pretty slow, but not too bad on a small scale
+//
+// For each item of the array, selection sort loops through the array again
+// to see if there are values less than the current minimum value.
+//
+// This is a very roundabout way to sort an array, but it works.
+package main
+
+import "fmt"
+
+// selectionSort returns a sorted array given arr, an array of ints.
+func selectionSort(arr []int) []int {
```

Comment:
This seems to be an 8-space tab. Can we narrow this down to be consistent with the rest.